packages/enketo-core/src/js/language.js

```diff
@@ -39,21 +41,27 @@ export default {
             this.form.view.html.querySelector('#form-languages');
 
         if (!formLanguages) {
-            return;
+            if (overrideLang) {
```

Comment:
Can you please add some comments here to walk through the update? For example, you can just say here that you're setting the default language to English if it's not defined in formLanguages.

Reply:
Done